tools/replay-viewer/style.css

```
@@ -0,0 +1,564 @@
+  line-height: 1.32;
+}
+
+.run-parameters__label,
+.legend-item {
+  color: #a8b9dc;
+}
+
+.run-parameters__value {
+  color: #f8fbff;
+  text-align: right;
+  word-break: break-word;
```

Stylelint (17.11.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)